bin/git-bc-show-eligible

```diff
@@ -40,14 +49,21 @@ def find_unpicked(repo, from_commit, to_commit, since_commit, show_all):
             break
 
 parser = argparse.ArgumentParser(description='Show commits, eligible for cherry-picking')
-parser.add_argument('branch', metavar='BRANCH', help='Name for the branch to check against')
+parser.add_argument('branch', metavar='BRANCH', help='Name for the branch to check against',
+                    default='master', nargs='?')
```

Comment:
I think `origin/master` is better. In case you are on a release branch, you do `git pull` and check the eligible commits. Many times `master` and `origin/master` are misaligned, and the latter seems to be a safer default.